modules/interactive_music/doc_classes/AudioStreamPlaybackPlaylist.xml

```diff
@@ -7,4 +7,12 @@
 	</description>
 	<tutorials>
 	</tutorials>
+	<methods>
+		<method name="get_playing_stream" qualifiers="const">
+			<return type="AudioStream" />
+			<description>
+				Returns the stream the playlist is currently playing.
```

Comment:
```suggestion
				Returns the stream the playlist that is currently playing.
```

Reply:
if "that" is needed, proper grammar would change it to "the stream _that_ the playlist is"

Reply:
or you would insert "from" to read "returns the stream from the playlist that is currently playing"

Reply:
It seems I have misunderstood the sentence, I thought it returns the whole playlist. In that case your first suggestion is better, yes

Reply:
Looking at it again idk what I was reading, it's not even ambiguous. Nevermind this

Reply:
No I get it, i think i should change it to be less vague since i can get how you'd misread it as returning the playlist and not the stream